nms/app/packages/magmalte/app/components/FormField.js

```diff
@@ -43,6 +44,15 @@ const useStyles = makeStyles(theme => ({
     marginRight: '15px',
     textAlign: 'right',
   },
+  subheading: {
+    fontWeight: '400',
+  },
+  optionalLabel: {
```

Comment:
Can you use the fonts and fontweight specified in magmalte/theme/default.js instead or if you feel we are missing something, can you make a change there and use it here. Thanks.

Reply:
The default styles there are already being used, and I can't just add more styles to the defaults there. It might take more time to do that part properly, would be better in a larger style design refactor PR later.